Bottom: 1e9e6d12529ab0c81b29da42322496e83ad8be93
Top:    792c237397202f0bc1247ea91ecb0214b4cfe708
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 01:57:53 +0100

Remove hack in draw.c which broke PS (and probably other) output

The set_layer call hacked into DrawLayerGroup to reset compositing tricked
the PS exporter to re-open and overwrite output in the main layer output
file during the fab drawing output.


---

diff --git a/src/draw.c b/src/draw.c
index 234e2ec..a7a1a74 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -695,9 +695,6 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
         rv = 0;
       if (layernum < max_copper_layer && Layer->On)
         dapi->draw_layer (Layer, drawn_area, NULL);
-
-      if (gui->gui) /* HACK: Subcomposite each layer in a layer group separately */
-        gui->set_layer (0, group, 0);
     }
   if (n_entries > 1)
     rv = 1;
